src/prof.c

```diff
@@ -1168,9 +1168,11 @@ prof_sample_threshold_update(prof_tdata_t *tdata) {
 	 */
 	r = prng_lg_range_u64(&tdata->prng_state, 53);
 	u = (double)r * (1.0/9007199254740992.0L);
-	tdata->bytes_until_sample = (uint64_t)(log(u) /
+	uint64_t bytes_until_sample = (uint64_t)(log(u) /
```

Comment:
Did we decide to allow mixed declarations and statements in our coding conventions?  (I think we did...)  If so, please refactor the declarations for `r` and `u` to make the function consistent.